editor/src/reveal/editor/reveal.clj

```diff
@@ -94,6 +100,13 @@
            :children #((:children %))
            :root (root-tree-node ec x)})))))
 
+(defn node-id-in-context
+  ([node-id]
+   (g/with-auto-evaluation-context evaluation-context
+     (node-id-in-context node-id evaluation-context)))
```

Comment:
It might be better if `reveal` just makes an evaluation-context but doesn't write to the cache?